Apps/Examples/Examples/All Examples/SymbolClusteringExample.swift

```diff
@@ -0,0 +1,166 @@
+import UIKit
+import MapboxMaps
+
+@objc(SymbolClusteringExample)
+
+public class SymbolClusteringExample: UIViewController, ExampleProtocol {
+
+    internal var mapView: MapView!
+
+    override public func viewDidLoad() {
+        super.viewDidLoad()
+
+        // Create a map view centered over Washington, DC.
+        let center = CLLocationCoordinate2D(latitude: 38.889215, longitude: -77.039354)
+        let cameraOptions = CameraOptions(center: center, zoom: 11)
+        let mapInitOptions = MapInitOptions(cameraOptions: cameraOptions, styleURI: .dark)
+        mapView = MapView(frame: view.bounds, mapInitOptions: mapInitOptions)
+        mapView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+
+        view.addSubview(mapView)
+
+        // Add the source and style layers once the
+        mapView.mapboxMap.onNext(.mapLoaded) { _ in
+            self.addSymbolClusteringLayers()
+        }
+        
+        let tap = UITapGestureRecognizer(target: self, action: #selector(handleTap(gestureRecognizer:)))
+        mapView.addGestureRecognizer(tap)
+    }
+
+    func addSymbolClusteringLayers() {
+        let style = self.mapView.mapboxMap.style
+        // The image named `fire-station-11` is included in the app's Assets.xcassets bundle.
+        let image = UIImage(named: "fire-station-11")!.withRenderingMode(.alwaysTemplate)
+        // Set `sdf` to `true`. This allows the icon images to be recolored.
+        try! style.addImage(image, id: "fire-station-icon", sdf: true)
+
+        // Fire_Hydrants.geojson contains information about fire hydrants in the District of Columbia.
+        // It was downloaded on 6/10/21 from https://opendata.dc.gov/datasets/DCGIS::fire-hydrants/about
+        guard let url = Bundle.main.url(forResource: "Fire_Hydrants", withExtension: "geojson") else {
```

Comment:
You should do this guard before the above let statements. Don't create vars if the guard will exit

Reply:
I would actually just convert this to a force unwrap and eliminate the guard. We have full control over the inputs so we know the file will always exist and this will never fail.